### ISSUE TITLE
Any chance of updating this to v1 and offering registration support?

### ISSUE THREAD

**Issue by tcatlas** [squez/Minecraft-Connect#2]
Would be really awesome to let users register on the forums using their Mojang/Minecraft account info.

**Comment by OLDsquez** [comment]
Yeah still working on it, haven't seen a lot of interest for the plugin so it's been dormant for awhile. Please post in the actual plugin thread! I don't check Github all that often.
